Dockerfile.template

```diff
@@ -21,38 +22,36 @@ RUN apt-get update && \
         x11-xserver-utils \
         xserver-xorg && \
     apt-get clean
+RUN apt-get -y remove stunnel4
```

Comment:
Why do we have this? If stunnel is installed already? Feels a bit redundant.

Reply:
it was in an ansible script, and I thought it was some kind of precaution

Reply:
Ah, yeah i think that was legacy. You should be able to remove that:

```
$ grep 'stunnel' ./* -R
./ansible/roles/ssl/tasks/main.yml:- name: Removes stunnel
./ansible/roles/ssl/tasks/main.yml:    name: stunnel4
```